eng/package.ps1

```diff
@@ -9,10 +9,10 @@ Write-Host "MSBUILD_EXE: $env:MSBUILD_EXE"
 
 $artifacts = Join-Path $PSScriptRoot ../artifacts
 $logsDirectory = Join-Path $artifacts logs
-if ($IsWindows) {
-    $sln = Join-Path $PSScriptRoot ../Microsoft.Maui.Packages.slnf
-} else {
+if ($IsMacOS) {
```

Comment:
Just an idea, but can we just delete this powershell file? Why do we need any powershell files if we are using cake?

A top-level `build.ps1` seems like it would be the only useful/needed powershell file.

Reply:
I think is that one wound't really only on cake. But since we are all in, I guess I m fine with it.

Reply:
Would be nice to delete either all the `.cake` files or all the powershell. No reason to have both.

Reply:
removing now... will see why we had it before...

Reply:
seemed to work!